src/overlays/actors/ovl_En_Rail_Skb/z_en_rail_skb.c

```diff
@@ -270,8 +270,8 @@ void EnRailSkb_Init(Actor* thisx, PlayState* play) {
 
     func_80B708C0(this, play);
     ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 36.0f);
-    SkelAnime_Init(play, &this->skelAnime, &object_skb_Skel_005EF8, &object_skb_Anim_0064E0, this->jointTable,
-                   this->morphTable, 20);
+    SkelAnime_Init(play, &this->skelAnime, &gStalchildSkel, &gStalchildWalkAnim, this->jointTable, this->morphTable,
+                   20);
```

Comment:
```suggestion
    SkelAnime_Init(play, &this->skelAnime, &gStalchildSkel, &gStalchildWalkAnim, this->jointTable, this->morphTable,
                   STALCHILD_LIMB_MAX);
```
You could update all the instances of `SkelAnime_Init` in the actor as well.

Reply:
Updated all the SkelAnime_Inits in both files to use `STALCHILD_LIMB_MAX`.